projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -88,6 +88,12 @@
 			&[data-type='jetpack/field-consent'] {
 				align-self: center;
 			}
+
+			&:where( .wp-block-jetpack-contact-form .wp-block-separator.is-style-default ){
```

Comment:
Isn't the usage of `is-style-default` misses all the blocks with no style class, which means are the default?

Reply:
My understanding is that `is-style-default` is only applied when toggling off a block style selection in the editor. This selector, as it stands, would only then be targeting the block under those circumstances which would seem odd. 

In my earlier [comment](https://github.com/Automattic/jetpack/pull/40967#issuecomment-2608883714), I assumed we'd want to get rid of `is-style-default` from the selector:

>  I think if we're aiming to target the default block style (which is really the absence of a block style) we're best served by targeting the block class without the is-style-default

Reply:
I removed the `is-style-default` from the selector.